pyproject.toml

```diff
@@ -20,5 +20,5 @@ exclude = '''
 [tool.pytest.ini_options]
 testpaths = ["tests", "superdesk", "apps", "content_api"]
 python_files = "*_test.py *_tests.py test_*.py tests_*.py tests.py test.py"
-asyncio_mode = "strict"
+asyncio_mode = "auto"
```

Comment:
Why the change to `auto`? Pytest in Superdesk doesn't use the asyncio plugin for tests (instead we use the pytest provided one).
See the change [here](https://github.com/superdesk/superdesk-core/blob/async/.github/workflows/nose-tests.yml#L28)

Reply:
I see. I will fix it. And what's the reason we don't use it?